src/Enhavo/Bundle/UserBundle/DependencyInjection/Configuration.php

```diff
@@ -314,6 +314,7 @@ private function addConfigChangeEmailSection(NodeDefinition $node)
                                 ->variableNode('options')->defaultValue([])->end()
                             ->end()
                         ->end()
+                        ->scalarNode('auto_login')->defaultValue(true)->end()
```

Comment:
I would suggest to remove the auto login for change email at all. There is also no implementation that will check this configuration. Only reset password and registration should have auto login options, because both actions have the goal to login a user, change email is a feature for already logged in user.

Reply:
@gseidel there is implementation to autologin after change email confirm but the auto_login setting was not available before.
See ChangeEmailController line 149. So i will remove the implementation also.